tests/flytekit/unit/core/image_spec/test_default_builder.py

```diff
@@ -43,6 +43,7 @@ def test_create_docker_context(tmp_path):
             pip_secret_mounts=[(".gitconfig", '/etc/gitconfig'), ("secret_src_2", "secret_dst_2")],
             source_copy_mode=CopyFileDetection.ALL,
             copy=[tmp_file.relative_to(Path.cwd()).as_posix()],
+            builder_config={"uv_image": "uv_image_local","micromamba_image": "micromamba_image_local"},
```

Comment:
Can this be 
```suggestion
            builder_config={"uv_image": "uv_image_local"},
```

and we make sure that the `DEFAULT_MICROMAMBA_IMAGE` is in the dockerfile?

Reply:
OK, It seems better to test both specified and unspecified image.